tools/lsp/preview/wasm.rs

```diff
@@ -224,5 +224,12 @@ pub fn notify_diagnostics(diagnostics: &[slint_interpreter::Diagnostic]) -> Opti
 
 pub fn ask_editor_to_show_document(file: &str, selection: lsp_types::Range) {
     let Ok(file) = lsp_types::Url::from_file_path(file) else { return };
-    send_message_to_lsp(crate::common::PreviewToLspMessage::ShowDocument { file, selection })
+    send_message_to_lsp(crate::common::PreviewToLspMessage::ShowDocument { file, selection });
+}
+
+pub fn ask_lsp_to_add_component(
```

Comment:
This duplicated function seems not so useful. Calling `send_message_to_lsp` directly from the call site should be cleaner and simpler.